Firestore/core/src/firebase/firestore/immutable/sorted_map.h

```diff
@@ -17,6 +17,7 @@
 #ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_IMMUTABLE_SORTED_MAP_H_
 #define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_IMMUTABLE_SORTED_MAP_H_
 
+#include <type_traits>
```

Comment:
Did you mean to include `util/type_traits.h`?

Reply:
Leftover, sorry. I tried adding some `static_assert`s, but that didn't turn out to be very useful, because compilation attempts proceed after hitting a `static_assert`, drowning it in a pool of useless noise.